doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -4586,13 +4586,18 @@ disjunction.
 
      Lemma test a : P (a || a) -> True.
 
-  .. coqtop:: all undo
+  .. coqtop:: all
 
      move=> HPa; move/P2Q: HPa => HQa.
 
   or more directly:
 
-  .. coqtop:: all undo
+  .. coqtop:: none
+
+     Abort.
+     Lemma test a : P (a || a) -> True.
```

Comment:
Note that another option here is to use `Restart.`

Reply:
Isn’t `Restart` as ugly as `Undo`?

Reply:
It's not about ugliness I think, but rather about predictability.